pkg/storaged/utils.js

```diff
@@ -617,6 +630,30 @@ function get_children(client, path) {
     return children;
 }
 
+export function get_children(client, path) {
+    const children = get_children_for_teardown(client, path);
+
+    if (client.blocks[path]) {
+        const mdraid = client.blocks[path].MDRaidMember;
+        if (mdraid != "/")
+            children.push(mdraid);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18698-20230427-131407-22f0bf87-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)